src/smtpproto/protocol.py

```diff
@@ -297,20 +299,23 @@ def quit(self) -> None:
         """Send the QUIT command (required to cleanly shut down the session)."""
         self._send_command('QUIT')
 
-    def mail(self, sender: Union[str, Address]) -> None:
+    def mail(self, sender: Union[str, Address], smtputf8: bool = True) -> None:
         """
         Send the MAIL FROM command (starts a mail transaction).
 
         :param sender: the sender's email address
-
+        :param smtputf8: send the SMTPUTF8 option, if available on the server
         """
         self._require_state(ClientState.ready, ClientState.authenticated)
 
         args = []
         if '8BITMIME' in self._extensions:
             args.append('BODY=8BITMIME')
-        if 'SMTPUTF8' in self._extensions:
+        if smtputf8 and 'SMTPUTF8' in self._extensions:
```

Comment:
Let's visually separate the `if` blocks.
```suggestion

        if smtputf8 and 'SMTPUTF8' in self._extensions:
```